.github/workflows/pyright.yml

```diff
@@ -0,0 +1,57 @@
+name: Check types
+on:
+#  pull_request:
+  workflow_call:
+  workflow_dispatch:
+env:
+  NODE_VERSION: '18'
+  PYTHON_VERSION: '3.9'
+
+permissions:
+  contents: read
+
+jobs:
+  typecheck:
+    strategy:
+      matrix:
+        include: #ckan-image see https://github.com/ckan/ckan-docker-base, ckan-version controls other image tags
+          - ckan-version: "dev-v2.11"
```

Comment:
Shouldn't this use a tag instead of a branch?

Reply:
This seems prone to builds breaking due to upstream changes rather than anything we did in this repo. I don't like it.

Reply:
I was having difficulties with the docker container ckan installed version being found correctly